addon/components/inputs/select.js

```diff
@@ -116,81 +134,95 @@ export default AbstractInput.extend({
 
   // == Functions ==============================================================
 
-  isQueryDisabled (formValue) {
-    const bunsenId = this.get('bunsenId')
-    const modelDef = this._getModelDef()
-    return modelDef.query !== undefined && !utils.hasValidQueryValues(formValue, modelDef.query, bunsenId)
-  },
-
-  _getModelDef () {
-    const cellConfig = this.get('cellConfig')
-    const modelDef = this.get('bunsenModel')
-    const options = _.get(cellConfig, 'renderer.options')
+  /**
+   * Determine if query has references to properties that have yet to be set.
+   * If the query contains no references then there is nothing to wait on.
+   * @param {Object} formValue - form value
+   * @returns {Boolean} whether or not query is waiting on missing references
+   */
+  iswaitingOnReferences (formValue) {
```

Comment:
That's a much better name than `isQueryDisabled` but I'd prefer `isWaitingOnReferences`.